pre_commit/parse_shebang.py

```diff
@@ -20,7 +20,7 @@ def parse_filename(filename: str) -> tuple[str, ...]:
 
 
 def find_executable(
-        exe: str, _environ: Mapping[str, str] | None = None,
+        exe: str, *, _environ: Mapping[str, str] | None = None,
```

Comment:
if it's public it shouldn't be underscored. `env` is probably what it sounds be called to match subprocess

Reply:
done!